mmdet/datasets/samplers/infinite_sampler.py

```diff
@@ -48,7 +50,7 @@ def __init__(self,
         self.world_size = world_size
         self.dataset = dataset
         self.batch_size = batch_size
-        self.seed = seed if seed is not None else 0
+        self.seed = sync_random_seed(seed) if seed is not None else 0
```

Comment:
```suggestion
        self.seed = sync_random_seed(seed)
```

Setting the seed defaults to 0 will reduce randomness after resume. Better to generate a random number as the default seed.

Reply:
Currently the default value of seed is 0, so if the user does not set it, it will be 0.

Reply:
At present, the program will definitely pass the seed